conda_forge_tick/auto_tick.py

```diff
@@ -826,7 +835,7 @@ def _compute_time_per_migrator(mctx, migrators):
         else:
             num_nodes.append(len(mmctx.effective_graph.nodes))
 
-    num_nodes_tot = sum(num_nodes)
+    num_nodes_tot = max(sum(num_nodes), 1)
```

Comment:
Is this correct? Some migrations may have no nodes, like the migration migrator.

Reply:
This is to prevent a divide by zero in the next line. I will move it there.

Reply:
fixed